flow/environment.py

```diff
@@ -15,6 +15,7 @@
 import os
 import re
 import socket
+from functools import lru_cache
```

Comment:
The style checks failed here. We use a tool called pre-commit to manage code style, and we have configured a pre-commit hook that uses the package `isort` to make sure that import statements are sorted (instead of randomly arranged). If you look at the CI output, it says to move this down (it alphabetizes the statements `from collections...` and `from functools...`).

See this guide to install pre-commit: https://docs.signac.io/projects/flow/en/latest/support.html#set-up-a-development-environment

Reply:
Didn't catch what the diff was telling me, thanks!